client/src/pages/Configurations/ConfigBuild/EicrSectionReview.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from 'react';
+import Table from '../../../components/Table';
+import { DbConfigurationSectionProcessing } from '../../../api/schemas/dbConfigurationSectionProcessing';
+import { useUpdateConfigurationSectionProcessing } from '../../../api/configurations/configurations';
+import { useToast } from '../../../hooks/useToast';
+import { UpdateSectionProcessingEntryAction } from '../../../api/schemas';
+import { useApiErrorFormatter } from '../../../hooks/useErrorFormatter';
+
+/**
+ * EicrSectionReview displays an overview or review of eICR sections and allows
+ * users to choose an action for each section (retain, refine, remove).
+ * Radio inputs are fully accessible and can be selected by clicking anywhere
+ * in the containing table cell (td), supporting keyboard navigation as well.
+ */
+const EicrSectionReview: React.FC<{
+  configurationId: string;
+  sectionProcessing: DbConfigurationSectionProcessing[];
+}> = ({ configurationId, sectionProcessing }) => {
```

Comment:
I think it'd be ideal if we didn't make use of `React.FC` for the sake of consistency. Our typical approach seems to be:
```suggestion
interface EicrSectionReviewProps {
  configurationId: string;
  sectionProcessing: DbConfigurationSectionProcessing[];
}

export function EicrSectionReview({
  configurationId,
  sectionProcessing,
}: EicrSectionReviewProps) {
```